util/util.go

```diff
@@ -115,8 +81,8 @@ func getHandle() (*libHandle, error) {
 // This function is a wrapper around the libsystemd C library; if it cannot be
 // opened, an error is returned.
 func GetRunningSlice() (slice string, err error) {
-	var h *libHandle
-	h, err = getHandle()
+	var h *dlopen.LibHandle
```

Comment:
We'd need to expose to the user a `Close()` function and they'd need to make sure they close the handle after they're done with the functions.

With the journal is easier because it already has a life cycle of its own. Here we could add artificial `New/Close` functions. Is that what we want?